packages/legend-query/src/stores/filterOperators/QueryBuilderFilterOperator_IsEmpty.ts

```diff
@@ -46,11 +47,24 @@ export class QueryBuilderFilterOperator_IsEmpty extends QueryBuilderFilterOperat
     const propertyType =
       filterConditionState.propertyExpressionState.propertyExpression.func
         .genericType.value.rawType;
-    const multiplicity =
-      filterConditionState.propertyExpressionState.propertyExpression.func
-        .multiplicity;
+    let isOptional = false;
+    let propertyExpression: ValueSpecification | undefined =
```

Comment:
I think we can abstract this to an utility function and put that in `QueryBuilderOperatorsHelper`. Then both post-filter and filter logic can reuse that.